src/main/java/io/appium/java_client/pagefactory/README.md

```diff
@@ -0,0 +1,75 @@
+# Standard Selectors
+
+## AndroidFindBy / iOSXCUITFindBy / WindowsFindBy
+
+# Advanced Selectors
+
+## iOS's Class Chain Queries
+
+Our XCUiTest integration has full support for the 'Class Chain' concept. This
+can do much of what XPath does...but faster.
+
+### External References
+
+Refer to [the official WebDriverAgent query docs](https://github.com/facebookarchive/WebDriverAgent/wiki/Class-Chain-Queries-Construction-Rules)
+to learn more about the general concept.
+
+### Sample usage:
+
+```java
+// Selector for image elements
+@iOSXCUITFindBy(iOSClassChain = "**/XCUIElementTypeImage")
+
+// Selector for every cell with the name 'Foo'
+@iOSXCUITFindBy(iOSClassChain = "**/XCUIElementTypeCell[`name == \"Foo\"`]")
```

Comment:
NSPredicate allows to use both double and single quotes in the equal manner. In java strings we could use the single ones to avoid unnecessary escaping.

Reply:
one should also be careful if the quoted string is a variable, which can also contain the same type of quotes that are used in the predicate. In such case this would be a developer's responsibility to take care quotes escaping inside the string variable. See, for example: https://stackoverflow.com/questions/14116217/nspredicate-single-quote-issue

Reply:
Good points. I'll try to present one or two more examples with different quoting styles.